src/module/script.js

```diff
@@ -0,0 +1,21 @@
+import { updateTaskIndexes, renderTaskList } from './index.js';
+
```

Comment:
Good work implementing modules! 💯 

- I've been trying to delete a task, but when I check it, the local storage stop working and there's no data saved, and the list of tasks just disappears. So let's fix this issue please 💪🏼 

- Kindly fix the problem when there's a task with the same name, there's no possibility to delete only the one with a checkbox. It should delete only the task selected, not all with the same name